### PR TITLE
Update svgjs 3.1.1

### PR DESCRIPTION
**Pull request by Rotzbua**
* Update to latest svgjs version 3.1.1
* Change cross to be more visible #100
  * color taken from #101 + border for better detection
  * increased size
* Convert quotes to just single quotes

### CODE REVIEW

**Comment by pardoman** [comment]
Why convert quotes? adds changes everywhere, for no apparent benefit.

**Comment by Rotzbua** [comment]
Because mixed quotes are disturbing.

File: js/main.js [hunk 1]
```diff
@@ -155,47 +161,54 @@ function displayData() {
...
                     })
-                    .mouseout(resetHighlightPath)
+                    .mouseout(function () {
+                        resetHighlightPath();
+                    })
                     .click(function () {
                         displayHelp(caret.id);
                     });
-
+
```
Comment (simonsan) [inline js/main.js:211]:
```suggestion

```

File: js/main.js [hunk 2]
```diff
@@ -852,14 +864,14 @@ function getInitialLocale() {
 function main() {
 
     history.pushState = (f => function pushState() {
-        const ret = f.apply(this, arguments);
+        const ret = f.apply(this, arguments); // Fixme: Void function return value is used.
```
Comment (simonsan) [inline js/main.js:867]: Fixme: Void function return value is used.

File: js/main.js [hunk 3]
```diff
@@ -852,14 +864,14 @@ function getInitialLocale() {
...
         window.dispatchEvent(new Event('pushstate'));
         window.dispatchEvent(new Event('locationchange'));
         return ret;
     })(history.pushState);
 
     history.replaceState = (f => function replaceState() {
-        const ret = f.apply(this, arguments);
+        const ret = f.apply(this, arguments); // Fixme: Void function return value is used.
```
Comment (simonsan) [inline js/main.js:874]: Fixme: Void function return value is used.

File: js/techtree.js [hunk 4]
```diff
@@ -445,7 +445,7 @@ class Lane {
             for (let i = 0; i < this.rows[r].length; i++) {
                 this.rows[r][i].y = offsets[r];
                 this.rows[r][i].x = x;
-                this.rows[r][i].width = element_height;
+                this.rows[r][i].width = element_height; // Todo: check warning `lane_width`?
```
Comment (simonsan) [inline js/techtree.js:448]: Todo: check warning `lane_width`

Comment (HSZemi) [reply js/techtree.js:448]: It's a square :upside_down_face:

File: js/main.js (hunk 2 above)
Comment (Rotzbua) [reply js/main.js:866]: It is old code. The warning is from the IDE but seems valid so I added the comment. Better documented than ignored.

File: js/techtree.js (hunk 4 above)
Comment (Rotzbua) [reply js/techtree.js:448]: IDE showed warning. Was not sure so I wrote it down.

Comment (HSZemi) [reply js/techtree.js:448]: Renamed it to `element_length` :smile:

**Comment by HSZemi** [comment]
Many thanks for the pull request! :yellow_heart: 
For now I opted to use the wooden cross from DE instead of tinkering further with the svg cross to improve visibility.
Therefore I excluded that change from your commits, and also fixed an issue with the quotation marks replacement that broke the stats display.
The changes have been added in commits 45be01c279caf1d81e65a6fe35ccd047f1873520 and 0002a51a52ce78e4516ecc4d9cbc0f99017b3858

**Pull request closed without merging**